.gitpod.Dockerfile

```diff
@@ -1 +1,9 @@
+#
+# Copyright © 2024, Ozone HIS <info@ozone-his.com>
```

Comment:
We need to correct the inception year; it's not 2024. Please verify the year of the first commit. I recommend removing this change from the current PR and creating a separate PR specifically to fix the license header.

Reply:
Updated it to 2021.
I think we are already changing almost all the files maybe we can consider adding this as well. WDYT @rbuisson 

![Screenshot 2024-07-05 at 12 10 41 AM](https://github.com/ozone-his/eip-odoo-openmrs/assets/36266597/8ff7152a-2bf2-4d54-8e1f-186029a98252)